Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py

```diff
@@ -184,12 +194,31 @@ def test_fetch_incidents(mocker):
     Then:
         - Verify the last run is set as the created time + 1 millisecond, i.e. 2020-03-22T13:22:13.934Z
     """
+    from AWS_SecurityHub import fetch_incidents
     mocker.spy(demisto, 'setLastRun')
     client = MockClient()
     fetch_incidents(client, 'Low', False, None, 'Both', None, None, None)
     assert demisto.setLastRun.call_args[0][0]['lastRun'] == '2020-03-22T13:22:13.934000+00:00'
 
 
+@freeze_time("2022-05-03")
+def test_fetch_with_archive_findings_without_findings(mocker):
+    """
+    Given:
+        - fetch incident with archive_findings parameter set to true.
+    When:
+        - Fetching finding as incident
+    Then:
+        - Verify that the fetch function terminate without errors.
+    """
+    from AWS_SecurityHub import fetch_incidents
+    mocker.patch.object(demisto, "getLastRun", return_value={})
+    set_last_run_mocker = mocker.spy(demisto, 'setLastRun')
+    client = MockClient(return_findings=False)
+    fetch_incidents(client, 'Low', True, None, 'Both', None, None, None)
+    assert set_last_run_mocker.call_args[0][0]['lastRun'] == '2022-04-18T00:00:00+00:00'
```

Comment:
Why wont add assertions to test the error we got in the bug? More than testing the last run object? For example, that no exception has been raised.

Reply:
added